lib/watir/elements/image.rb

```diff
@@ -16,18 +16,6 @@ def loaded?
       )
     end
 
-    #
-    # Returns the image's width in pixels.
-    #
-    # @return [Integer] width
-    #
-
-    def width
```

Comment:
Actually, for the Elements that include a valid attribute for `width` (`Table`, `Input`, etc), it matches the 'y' value returned by `WebDriver::Element#size`. The Selenium call can be made on any element, though, so I implemented the code to always use it instead of `#attribute_value`.